src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -225,12 +225,12 @@ class BinaryCompactObject : public DomainCreator<3> {
     static type default_value() noexcept { return false; }
   };
 
-  struct OuterSphericalShellLayers {
+  struct OuterLayerRadialRefinements {
     using type = size_t;
     static constexpr OptionString help = {
-        "The number of radial layers in the outer sphere."};
-    static constexpr type default_value() noexcept { return 1; }
-    static type lower_bound() noexcept { return 1; }
+        "The number of radial refinements in the outer layer (layer 4)."};
```

Comment:
It's no longer clear that `outer layer` is the same as `Layer 4` (the way this documentation is written). I think it might be confusing to readers that what is currently the `outer layer` and `Layer 4` can refer to up to 20 blocks, and visually is better described as two layers of blocks. (The reader might think `outer layer` refers to only the outermost ten blocks, which isn't all of Layer 4 if 20 are present.) I'm going to suggest the following changes:
In the documentation:
```
 * - 4: The 10 blocks that form the first outer shell. This layer transitions back to
 *      spherical. The gridpoints are distributed linearly with respect to radius.
 * - 5: (optionally) The 10 blocks that form an additional outer shell. This layer is
*      entirely spherical, so a logarithmic map can be used in this layer. This allows
*      the domain to extend to large radial distances from the compact objects.
 
```

And perhaps, if you agree with it:
```
* `enveloping cube` and `enveloping sphere` refer to the outer surfaces of
 * layers 3 and 4 respectively. Both of these surfaces are centered at the
 * origin.
 * `outer layer` refers to Layers 4 and 5 (if present) together. 
 * `cutting plane` refers to the plane along which the domain divides into two
 * hemispheres. In the final coordinates, the cutting plane always intersects
 * the x-axis at the origin.
```

Having defined Layer 5 it makes it easier for me to recommend the following changes, because I think this option is really trying to do the job of two separate but related objects, the presence of Layer 5 and the refinement level of Layer 5:
`OuterLayerRadialRefinements` can be changed to `AdditionalRadialRefinement` and line 333 in the cpp can be changed to:

```
 gsl::at(initial_levels[block], 2) += additional_radial_refinement_;
```
because I think the presence of the `- 1` otherwise is a bit clunky and makes the interpretation of `OuterLayerRadialRefinements` possibly confusing.

The options can be:
```
struct AdditionalRadialRefinement {
    using type = size_t;
    static constexpr OptionString help = {
        "The additional radial refinement applied to Layer 5."};
    static constexpr type default_value() noexcept { return 0; }
    static type lower_bound() noexcept { return 0; }
};

```
```
struct AddOuterSphericalShell {
    using type = bool;
    static constexpr OptionString help = {
        "Use Layer 5, allowing the log spaced radial grid to be used."};
    static constexpr type default_value() noexcept { return false; }
};
```

Reply:
How about AddOuterSphericalShell and OuterSphericalShellRefinement?

Reply:
If the `+=` is still used in line 333, I prefer the use of `Additional`, if not in the option name at least in the help text. Maybe `AddOuterShellRefinement` ?

Reply:
I'm not clear on why the outer shell is being considered as something optional. When would we ever not want a wave zone in CBC simulations?

Reply:
If it's not optional that would simplify things - I was thinking leaving it optional in terms of allowing for fewer blocks to be used as well as this being a testing domain, but if there really is no reason to test such a domain as it'll never be used then that's a good point.

Reply:
It was optional to exactly reproduce the domain as originally coded, but I agree that this is not necessary.

Reply:
Okay, sounds like we're all okay making the wave zone required. Let's do that then, since I think it'll simplify the code and will always be what we want. If someone wants a small wave zone they can just make the outer radius close :)